bootstrap

```diff
@@ -2,14 +2,25 @@
 %% -*- mode: erlang;erlang-indent-level: 4;indent-tabs-mode: nil -*-
 %% ex: ft=erlang ts=4 sw=4 et
 
-main(_) ->
+main(Args) ->
+    case lists:member("--offline", Args) of
+        true ->
+            os:putenv("REBAR_OFFLINE", "1");
+        false ->
+            _
```

Comment:
Change `_` to `ok` here

Reply:
Yes of course, this code doesn't work. I can fix it if there is a chance that this idea will be accepted at all.